### ISSUE TITLE
bootblog4 looks meh on mobile

### ISSUE THREAD

**Issue by Kwpolska** [getnikola/nikola#3069]
http://ralsina.me/

**Comment by Kwpolska** [comment]
cc @ralsina

**Comment by ralsina** [comment]
Yes, it is not the prettiest. I suspect the search form needs tweaking, since it was bootstrap3 code originally.

**Comment by ralsina** [comment]
The search form should collapse into the menu, that would make it look ok. I am trying to do it but ... eek CSS.

**Comment by ralsina** [comment]
Blah, the example doesn't even have a search form, just a fake thing.
